.github/actions/mint-token/action.yml

```diff
@@ -17,6 +17,9 @@ inputs:
   requested_permissions:
     description: 'Permission request information in the form {"repositories":["github-token-minter"],"permissions":{"issues":"read"}}'
     required: true
+  use_gcloud_token:
```

Comment:
This seems like the incorrect way to use this. We should always use the GitHub id_token from a GitHub action (the identity is the workflow).

Instead the token would be coming from a service running on GCP. I think your integration test should instead generate an id_token for a service account and send it to the minty endpoint. See comment below.

Reply:
I agree. We shouldn't be modifying the GitHub action at all. You can turn this test into a simple `curl` with the token from GCP in the correct header.

Reply:
Moved the test into `ci.yml` as a couple of `curl` calls with the proper tokens.